README.md

```diff
@@ -1,16 +1,58 @@
-This directory contains the programs needed to generate and verify SigStore root keys and create signed TUF metadata. 
+This directory contains the programs needed to generate and verify Sigstore root keys and create signed TUF metadata. 
+
+## TUF Repository Structure
+
+The current published repository metadata lives in the [repository](/repository/repository) subfolder of this GitHub repository. In this repository, you will find the top-level TUF metadata files, delegations, and target files. 
+
+* [root.json](repository/repository/root.json): This is the current `root.json`. It is signed by at least 3 out of the 5 [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders). Other signing keys endorsed by this root include:
+  * A set of 5 targets keyholders (the root keyholders).
```

Comment:
nit: Mention why we chose the same set of target keyholders as root keyholders? Something around that both are offline keys so we wanted to avoid managing multiple offline keysets?